liri.js

```diff
@@ -1,186 +1,157 @@
 // get input from user and set it as an array while ignoring first 2 index's
 var inputArr = process.argv.slice(3);
+var Twitter = require('twitter');
+var dataArr = [];
+var Spotify = require('node-spotify-api');
+var request = require('request');
+var fs = require('fs');
+var keys = require("./keys.js");
+
+// twitter function
+var twitterFunc = function() {
+	var twitter = new Twitter ({
+	  	consumer_key: keys.twitterKeys.consumer_key,
+	  	consumer_secret: keys.twitterKeys.consumer_secret,
+	  	access_token_key: keys.twitterKeys.access_token_key,
+	  	access_token_secret: keys.twitterKeys.access_token_secret
+	});
```

Comment:
I think better practice would be to create your twitter instance in your keys file and then export and require that instance in your liri file.